hw/top_englishbreakfast/data/top_englishbreakfast.hjson

```diff
@@ -226,9 +226,14 @@
     },
     { name: "pwrmgr_aon",
       type: "pwrmgr",
```

Comment:
does it make sense to carry over the other config changes as well?

Reply:
ah i didn't bother for englishbreakfast since it doesn't have an alert handler.
is that what you meant?